contributing/deployment-tools.txt

```diff
@@ -0,0 +1,148 @@
+OME deployment tools
+====================
+
+This section describes deployment tools supported by the OME team.
+It is primarily designed for the core OME developers who want to
+bring new or upgrade existing prerequisites. The following steps explain
+the connections between basic repositories and the testing workflow.
+
+
+.. note::
+
+    This section requires brief understanding of ansible 
+    http://docs.ansible.com/ansible/intro_getting_started.html
+    and Docker engine https://docs.docker.com/.
+
+Prerequisites locations
+-----------------------
+
+List of OME prerequisites is stored in multiple git repositories, each of 
+which is available from several locations.
+
+OME Infrastructure (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Infrastructure is provided to simplify deployment using 
+`Ansible <https://www.ansible.com>`.
```

Comment:
This needs some final underscore to render correctly (or a top level `.. _Ansible: https://www.ansible.com` so that `Ansible_` can be used.)